dss_datamover/master_application.py

```diff
@@ -425,10 +447,15 @@ def start_indexing(self):
                     self.logger.info('start_indexing: Processing prefixes for indexing stopped')
                     break
                 self.logger.info("Processing prefix:{}".format(prefix))
-                (ip_address, nfs_share, ret) = self.nfs_cluster_obj.mount_based_on_prefix(prefix)
+                (ip_address, nfs_share, ret, out) = self.nfs_cluster_obj.mount_based_on_prefix(prefix)
+                if ret != 0:
+                    self.nfs_cluster_obj.umount_all()
+                    self.logger.error("NFS Mounting failed, Error: \n**{}**".format(out))
```

Comment:
why are you adding newline \n. Not acceptable. Also remove *'s as well

Reply:
updated in latest commit.